GliaWidgets/SecureConversations/Welcome/SecureConversations.MessageTextView.swift

```diff
@@ -0,0 +1,318 @@
+import UIKit
+
+extension SecureConversations.WelcomeView {
+    class MessageTextView: BaseView {
+        enum Props: Equatable {
+            struct NormalState: Equatable {
+                let style: SecureConversations.WelcomeStyle.MessageTextViewStyle
+                let text: String
+                let activeChanged: Command<Bool>
+            }
+
+            struct ActiveState: Equatable {
+                let style: SecureConversations.WelcomeStyle.MessageTextViewStyle
+                let text: String
+                let textChanged: Command<String>
+                let activeChanged: Command<Bool>?
+            }
+
+            struct DisabledState: Equatable {
+                let style: SecureConversations.WelcomeStyle.MessageTextViewStyle
+                let text: String
+            }
+
+            case normal(NormalState)
+            case active(ActiveState)
+            case disabled(DisabledState)
+        }
+
+        let environment: Environemnt
+
+        var props = Props.normal(.init(style: .initial, text: "", activeChanged: .nop)) {
+            didSet {
+                renderProps()
+            }
+        }
+
+        lazy var textView = UITextView().makeView { textView in
+            textView.backgroundColor = .clear
+            textView.delegate = self
+        }
+
+        lazy var placeholderLabel = UILabel().makeView { label in
+            label.numberOfLines = 0
+        }
+
+        var textViewTopConstraint: NSLayoutConstraint?
+        var textViewBottomConstraint: NSLayoutConstraint?
+        var textViewLeadingConstraint: NSLayoutConstraint?
+        var textViewTrailingConstraint: NSLayoutConstraint?
+
+        init(environment: Environemnt) {
+            self.environment = environment
+            super.init()
+        }
+
+        required init() {
```

Comment:
Could you add `unavailable` clause?

Reply:
fixed [9e81485](https://github.com/salemove/ios-sdk-widgets/pull/976/commits/9e814856472c73aee93ed97a6e6cf7433e285210)